site/src/components/Popover/Popover.tsx

```diff
@@ -141,7 +141,8 @@ export const PopoverContent: FC<PopoverContentProps> = ({
     // disablePortal, it is using aria-hidden="true". It is a bug but it is not
     // fixed yet so we have to define our own container for now.
     // https://github.com/mui/material-ui/issues/19450#issuecomment-1539832475
-    <div ref={containerRef}>
+    // Position is absolute so it does not influence flex layout as an extra element
+    <div ref={containerRef} css={{ position: "absolute" }}>
```

Comment:
I'm a little confused by the change. It looks like the main thing that's happening is that this is making sure the element gets removed from the CSS flow and can't affect other elements

That seems like a good change – I'm just not sure how it's connected to accessibility

Reply:
I missed your comment but this is because MUI uses aria-hidden=true on the default container for some reason. A few ppl are discussing about that in the git issue that I put in the comments

Reply:
At some point, we can revisit it and remove this workaround